package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vets-json-schema",
-  "version": "19.0.1",
+  "version": "19.0.2",
```

Comment:
You'll need to pull the latest changes from `master` and increment the version after doing so.